backend/prisma/seed.js

```diff
@@ -1,73 +1,68 @@
 //const {events} = require('./eventData')
-const { PrismaClient } = require("@prisma/client");
-const prisma = new PrismaClient();
+const { PrismaClient } = require('@prisma/client')
+const prisma = new PrismaClient()
+
 
 const events = [
-  {
```

Comment:
There are a lot of code format changes in this PR, which can bury the real important changes. In future PRs, let's split the no-op changes with the real ones, so ppl know what to look at.

Reply:
Done